eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -87,11 +89,19 @@ else {
     # setting variable to reference the pull request by number
     Write-Host "##vso[task.setvariable variable=Submitted.PullRequest.Number]$($resp.number)"
 
-    Update-GitHubIssue -RepoOwner $RepoOwner -RepoName $RepoName -IssueNumber $resp.number `
-    -Labels $PRLabels -Assignees $Assignees -AuthToken $AuthToken
-
     Add-GitHubPullRequestReviewers -RepoOwner $RepoOwner -RepoName $RepoName -PrNumber $resp.number `
     -Users $UserReviewers -Teams $TeamReviewers -AuthToken $AuthToken
+
+    if ($IsTest) {
+      $prState= "closed"
+      LogDebug "Updating https://github.com/$RepoOwner/$RepoName/pull/$($resp.number) state to closed"
```

Comment:
```suggestion
      LogDebug "Updating https://github.com/$RepoOwner/$RepoName/pull/$($resp.number) state to closed because this was only testing."
```